UPGRADE.md

```diff
@@ -100,3 +100,82 @@
    +                       class: "%sylius.model.channel.class%"
    #           ...
    ```
+   
+1. Service and Alias Refactoring for [New Naming Convention](https://github.com/Sylius/Sylius/blob/2.0/adr/2024_10_03_services_naming_convention.md)
+   Including the following changes:
+
+#### Root configuration key
+
+   ```diff
+      - sylius_invoicing_plugin:
+      + sylius_invoicing:
+   ```
+
+#### Routing
+
+| Old route                                            | New route                                     |
+|------------------------------------------------------|-----------------------------------------------|
+| sylius_invoicing_plugin_invoice                      | sylius_invoicing_invoice                      |
+| sylius_invoicing_plugin_admin_invoice_show           | sylius_invoicing_admin_invoice_show           |
+| sylius_invoicing_plugin_admin_invoice_download       | sylius_invoicing_admin_invoice_download       |
+| sylius_invoicing_plugin_admin_invoice_resend         | sylius_invoicing_admin_invoice_resend         |
+| sylius_invoicing_plugin_admin_order_invoices_partial | sylius_invoicing_admin_order_invoices_partial |
+| sylius_invoicing_plugin_shop_invoice_download        | sylius_invoicing_shop_invoice_download        |
+| sylius_invoicing_plugin_shop_order_invoices_partial  | sylius_invoicing_shop_order_invoices_partial  |
+| sylius_invoicing_plugin_admin                        | sylius_invoicing_admin                        |
+| sylius_invoicing_plugin_shop                         | sylius_invoicing_shop                         |
+
+#### Grids
+
+| Old grid                        | New grid                 |
+|---------------------------------|--------------------------|
+| sylius_invoicing_plugin_invoice | sylius_invoicing_invoice |
+
+#### Changed Services
+
+| Old id                                                                             | New id                                                                      |
+|------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|
+| sylius_invoicing_plugin.event_listener.workflow.payment.produce_order_payment_paid | sylius_invoicing.event_listener.workflow.payment.produce_order_payment_paid |
+| sylius_invoicing_plugin.ui.action.download_invoice                                 | sylius_invoicing.ui.action.download_invoice                                 |
+| sylius_invoicing_plugin.ui.action.resend_invoice                                   | sylius_invoicing.ui.action.resend_invoice                                   |
+| sylius_invoicing_plugin.cli.generate_invoices                                      | sylius_invoicing.cli.generate_invoices                                      |
```

Comment:
In `Sylius/Sylius`, we moved CLI commands to the `Console/Command` namespace. Should we do same here?

Reply:
For consistency would be good, but definitely not in this PR and perhaps not before the next releases 🤔